### PR TITLE
Whitespace removal and formatting

### PR DESCRIPTION
**Pull request by PalmEmanuel**
## Description

Removed unnecessary whitespace, and adjusted formatting for a large portion of the Radius docs, excluding CLI reference which is in PR #978.

Related to previous PR https://github.com/radius-project/docs/pull/967 which was asked to be split.

### Summary
:art::memo::wastebasket:

This pull request contains formatting improvements for various documentation and code files in the `radius-project/docs` repository. The main changes are removing unnecessary or trailing whitespace, adding blank lines between code blocks, and cleaning up code formatting. These changes aim to enhance the readability and quality of the documentation and code snippets.

> _Oh we are the coders of the sea_
> _And we like our files neat and tidy_
> _So we pull and we push and we trim the whitespace_
> _On the count of one, two, three_

### Walkthrough
*  Remove trailing whitespaces from various lines in `docs/assets/scss/_nav.scss` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-d8530aa7441be66e6f7d29b64da90a3a82463460cc3244b349e43fb239836ef6L125-R130),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-d8530aa7441be66e6f7d29b64da90a3a82463460cc3244b349e43fb239836ef6L168-R169),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-d8530aa7441be66e6f7d29b64da90a3a82463460cc3244b349e43fb239836ef6L192-R214))
*  Remove extra blank lines between CSS rules in `docs/assets/scss/_nav.scss` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-d8530aa7441be66e6f7d29b64da90a3a82463460cc3244b349e43fb239836ef6L139-R144),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-d8530aa7441be66e6f7d29b64da90a3a82463460cc3244b349e43fb239836ef6L150-R150))
*  Add blank lines between sections in `docs/content/community/contributing/contributing-docs/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-67bd8967b394fb430efd51387bf2dfae7d9d611d2494ba954c6f30e92d33fe35R149))
*  Remove blank line between code blocks in `docs/content/community/contributing/contributing-docs/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-67bd8967b394fb430efd51387bf2dfae7d9d611d2494ba954c6f30e92d33fe35L347))
*  Remove trailing whitespaces from various lines in `docs/content/community/contributing/contributing-docs/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-67bd8967b394fb430efd51387bf2dfae7d9d611d2494ba954c6f30e92d33fe35L594-R594))
*  Remove trailing whitespaces from various lines in `docs/content/concepts/api-concept/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16dce7b693eaf35cc11f9e19b3e6a50b9a2fb845992e6edbd862eebe801eae5bL30-R33),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16dce7b693eaf35cc11f9e19b3e6a50b9a2fb845992e6edbd862eebe801eae5bL41-R45),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16dce7b693eaf35cc11f9e19b3e6a50b9a2fb845992e6edbd862eebe801eae5bL65-R67))
*  Remove trailing whitespaces from various lines in `docs/content/concepts/collaboration-concept/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-c8dcec5e1382cf1e109719b54936fd4a4eae04c92c11740a98609420e8e718e9L20-R22))
*  Remove blank line between paragraphs in `docs/content/concepts/collaboration-concept/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-c8dcec5e1382cf1e109719b54936fd4a4eae04c92c11740a98609420e8e718e9L38-R37))
*  Remove trailing whitespaces from various lines in `docs/content/concepts/faq/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-69df4f813a3c3e019a80a4a10de4aaaf256f9699048b08cd9383c1985a6d73d4L11-R11),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-69df4f813a3c3e019a80a4a10de4aaaf256f9699048b08cd9383c1985a6d73d4L19-R23),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-69df4f813a3c3e019a80a4a10de4aaaf256f9699048b08cd9383c1985a6d73d4L48-R48),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-69df4f813a3c3e019a80a4a10de4aaaf256f9699048b08cd9383c1985a6d73d4L56-R58),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-69df4f813a3c3e019a80a4a10de4aaaf256f9699048b08cd9383c1985a6d73d4L66-R76))
*  Remove trailing whitespaces from various lines in `docs/content/concepts/overview/snippets/appmodel-concept.bicep` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-5a021c958ea0c180414f6a91e6b53d693d5db0236db8212e7cda12a664c9e5bbL34-R34))
*  Remove trailing whitespaces from various lines in `docs/content/concepts/why-radius-concept/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-969ad269838b1687dd34dac9e4e92807eb60ee9b9ac81fa9f33ac184cc101dfcL46-R50))
*  Remove trailing whitespaces from various lines in `docs/content/getting-started/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16c6b1f038bfac9806df34f162b60508f10bfe6eb84a7b62f2a1d7c8d83fd23cL49-R49),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16c6b1f038bfac9806df34f162b60508f10bfe6eb84a7b62f2a1d7c8d83fd23cL58-R72),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-16c6b1f038bfac9806df34f162b60508f10bfe6eb84a7b62f2a1d7c8d83fd23cL178-R178))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/application/overview/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-2c291895e3da01a4e73f30a7eac90ec51bb9e6141a0e91226022d27d4257ea22L41-R41))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/aws/howto-aws-resources/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5L16-R25),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5L31-R34),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5L82-R85),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5L129-R132))
*  Add blank lines between code blocks in `docs/content/guides/author-apps/aws/howto-aws-resources/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5R115),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e3c58ec3e1010335fbfe110c807c9220c058a2430e619feff2ba56a34909b4f5R125))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/aws/overview/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-58755cda8fd231e240d3ff4ce96af458d01bab5491924748308b92f1aa30aee5L24-R27),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-58755cda8fd231e240d3ff4ce96af458d01bab5491924748308b92f1aa30aee5L33-R33))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/aws/overview/snippets/aws.bicep` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e53cad9ad9d60e11d1d4d50a7437c8589cdc7036957f1f85db6398db9cf86801L10-R10))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/azure/howto-azure-resources/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-7cf8cf4bbc8df64f4103e1b30bad01ca70223c90c40f25f212bd1c8207246adcL5-R5),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-7cf8cf4bbc8df64f4103e1b30bad01ca70223c90c40f25f212bd1c8207246adcL27-R27),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-7cf8cf4bbc8df64f4103e1b30bad01ca70223c90c40f25f212bd1c8207246adcL35-R35))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/azure/howto-azure-resources/snippets/container-wi.bicep` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-de8e6eb6e7a98ae9ef00d19e6e88eb12aff02a1f0431e15b5d0380d7c7cf18a6L8-R8),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-de8e6eb6e7a98ae9ef00d19e6e88eb12aff02a1f0431e15b5d0380d7c7cf18a6L71-R71))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/containers/howto-connect-dependencies/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-0af862ed25c1639c118abf521442b6c2ca7999320b0221b3dbdc8033d16882feL44-R48))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/containers/howto-keyvault-volume/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-5c8c193aa4b080d7d4a05f80533a3ae96c6dc933b55129f1e79110447ffd5e13L5-R5),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-5c8c193aa4b080d7d4a05f80533a3ae96c6dc933b55129f1e79110447ffd5e13L26-R28),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-5c8c193aa4b080d7d4a05f80533a3ae96c6dc933b55129f1e79110447ffd5e13L76-R76),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-5c8c193aa4b080d7d4a05f80533a3ae96c6dc933b55129f1e79110447ffd5e13L85-R85))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/containers/howto-keyvault-volume/snippets/keyvault-wi.bicep` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-083a5017049e55095b33a738d2462f8f9007da62e252be6acdbfc13f8d371863L87-R94))
*  Add blank lines between code blocks in `docs/content/guides/author-apps/containers/howto-volumes/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-6a99da3eff5d87692a3b8f6b7019ca7b3f902d2b73815d5dca70e0bd458bf7a6R37),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-6a99da3eff5d87692a3b8f6b7019ca7b3f902d2b73815d5dca70e0bd458bf7a6R43))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/containers/howto-volumes/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-6a99da3eff5d87692a3b8f6b7019ca7b3f902d2b73815d5dca70e0bd458bf7a6L65-R67))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/containers/overview/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-fbd5baae25af2b349d637a6600c0f938d63bb40fa0f8835d6a761e9c2fe48400L33-R33),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-fbd5baae25af2b349d637a6600c0f938d63bb40fa0f8835d6a761e9c2fe48400L69-R69),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-fbd5baae25af2b349d637a6600c0f938d63bb40fa0f8835d6a761e9c2fe48400L95-R95),[link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-fbd5baae25af2b349d637a6600c0f938d63bb40fa0f8835d6a761e9c2fe48400L106-R106))
*  Remove trailing whitespaces from various lines in `docs/content/guides/author-apps/dapr/overview/index.md` ([link](https://github.com/radius-project/docs/pull/979/files?diff=unified&w=0#diff-e11137ae69895b2490ce5d89d4cb1513185e09fc1b45458bcd2117f24e487ce8L13-R13))

### CODE REVIEW

**Review by jasonviviano — APPROVED** [review]
Thanks again for separating the changes again! Looks great.

File: docs/content/tutorials/new-app/index.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Radius Applications are where all your app's resources and relationships come to
 
    {{% rad file="snippets/1-app.bicep" embed=true %}}
 
-1. Deploy `app.bicep` with [`rad deploy`]({{< ref rad_deploy >}}):
+2. Deploy `app.bicep` with [`rad deploy`]({{< ref rad_deploy >}}):
```
Comment (AaronCrawfis) [inline docs/content/tutorials/new-app/index.md]: Our markdown numbering purposefully uses `1.` for every item, as our markdown renderer automatically increments the number. Could you please keep them as `1.`?

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]
Thank you for the contribution! Left one note on our markdown numbering convention.

File: docs/content/tutorials/new-app/index.md (hunk 1 above)
Comment (PalmEmanuel) [reply docs/content/tutorials/new-app/index.md]: Changed them back to 1 for this  PR. There was some inconsistency across the docs in the numbering, but I'll change them to all to 1. in a separate PR. Thanks!

**Review by AaronCrawfis — APPROVED** [review]
Thanks for helping improve our docs!

**Pull request merged**